codacy/templates/fluentd/fluentd-conf.yaml

```diff
@@ -43,10 +43,10 @@ data:
       s3_object_key_format %{path}%{index}.%{file_extension}
 
       # We can't use this due to a bug in minio
-      # <bucket_lifecycle_rule>
-      #   id test-policy
-      #   expiration_days {{ .Values.fluentdoperator.expirationDays }}
-      # </bucket_lifecycle_rule>
+      <bucket_lifecycle_rule>
```

Comment:
don't we need to have minio bumped to have this working? I tought the version we were using had some kid on bug

Reply:
I tested this by hand in our sandbox environment, and it seems that it is working as expected